src/Raven.Client/Constants.cs

```diff
@@ -119,6 +119,16 @@ private Certificates()
             public const string Prefix = "certificates/";
             public const int MaxNumberOfCertsWithSameHash = 5;
         }
+        
+        public class Network
+        {
+            public const string AnyIp = "0.0.0.0";
+            public const int ZeroValue = 0;
+            public const int DefaultSecuredRavenDbHttpPort = 443;
+            public const int DefaultUnsecuredRavenDbHttpPort = 8080;
+            public const int DefaultSecuredRavenDbTcpPort = 38888;
+            public const int DefaultUnsecuredRavenDbTcpPort = 38881;
```

Comment:
I believe that unsecured tcp port is the same - 38888.

Reply:
I will check it again.
Regarding the `--help` of `-s` option, I think we will leave it for the documentation.
I did some research given the link you sent me and in any option, the JSON doesn't look good.
We can place it after the help text chuck but it does not seem to be the right thing to do.
Let's leave it for the documentation for now

Reply:
OK. What about putting it in an `rvn.md` text file next to it?

Reply:
@gregolsky  I removed it for now